ipc/provider/src/config/serialize.rs

```diff
@@ -98,16 +98,6 @@ mod tests {
     [server]
     json_rpc_address = "127.0.0.1:3030"
 
-    [[subnets]]
-    id = "/r123"
-    network_name = "test"
-
-    [subnets.config]
-    network_type = "fvm"
```

Comment:
I was planning to leave the FVM config for now, but I guess you are right and is better just to remove it

Reply:
yeah, it's breaking the tests and seems no point keeping.